src/Queue/Queue.php

```diff
@@ -189,4 +189,12 @@ private function checkConnected()
             $this->client->connect();
         }
     }
-}
\ No newline at end of file
+    
+    /**
```

Comment:
The method is public and should be located before all protected and private methods. Right now it is at the end, after a private method.
It is also missing a proper comment - `@inheritdoc` is not applicable, as the class `Queue` doesn't inherit or implement anything.